### DISCUSSION TITLE
initialFocusRef in Modal should open keyboard

### DISCUSSION THREAD

**Discussion by fafo** [GeekyAnts/NativeBase#5228] [Feature request]
### Description

When using initialFocusRef property in Modal, the keyboard should open, instead the cursor blinks the selected input, but you have to click on it in order to open the keyboard. 

### CodeSandbox/Snack link

https://snack.expo.dev/@faf0/64fa14

### Steps to reproduce

You can see it on the documentation example. Just click on open on a mobile device. The modal opens, the focus is on the correct input, but the keyboard closed

### NativeBase Version

Any above 3.2

### Platform

- [X] Android
- [ ] CRA
- [ ] Expo
- [ ] iOS
- [ ] Next

**Comment by fafo** [comment]
Sorry I put  temporary link to the snack. This one should be ok:
https://snack.expo.dev/@faf0/initial-focus-on-modal

**Comment by Krithikj20** [comment]
Hi @fafo, Thank you for reporting this issue. We will try to fix this at the earliest.

**Comment by Viraj-10** [comment]
Hi @fafo, `initialFocusRef` can't open the keyboard. In your case, it's `inputText` but Sometimes the user wants to focus on `radio` or `button` when Modal is open. So this doesn't align with the component library perspective. I am moving this to the discussion. We can have a conversation about it there.

**Reply by fafo** [reply]
Hi, I understand the doubt but in my understanding the correct behavior of .focus() on a text input in react native is to open the keyboard.
The issue here is that I see the cursor in the correct input, but the keyboard is closed. For this reason I thought of a bug, since it seems that something odd is happening while opening the modal and preventing the automatic opening of the keyboard

**Reply by Viraj-10** [reply]
got it, I guess only when input it focused should open the keyboard. seem a valid bug. Moving this back to issues. #5257
